dev/io.openliberty.microprofile.metrics.internal.5.0.monitor/src/io/openliberty/microprofile/metrics/internal/monitor/internal/MappingTable.java

```diff
@@ -0,0 +1,182 @@
+	public static final String SERVLET_TAG_NAME = "servlet";
+
+	public static final String CONNECTIONPOOL_TAG_NAME = "datasource";
+	public static final String SESSION_TAG_NAME = "appname";
+	public static final String JAXWS_SERVER_TAG_NAME = "endpoint";
+	public static final String JAXWS_CLIENT_TAG_NAME = "endpoint";
+
+	public static final String GRPC_SERVER_TAG_NAME = "grpc";
+	public static final String GRPC_CLIENT_TAG_NAME = "grpc";
+	
+	public static final String COUNTER = "COUNTER";
+	public static final String GAUGE = "GAUGE";
+
+	private static MappingTable singleton = null;
+
+	private Map<String, String[][]> mappingTable = new HashMap<String, String[][]>();
+
+	public static MappingTable getInstance() {
+		if (singleton == null)
+			singleton = new MappingTable();
+		return singleton;
+	}
+	
+	
+	private MappingTable() {
+		
+		String[][] requestTimeTable = new String[][] {
+			{ "vendor", "requestTiming.requestCount", "Request Count", "requestTiming.requestCount.description", COUNTER, MetricUnits.NONE, "RequestCount", null, null },
+			{ "vendor", "requestTiming.activeRequestCount", "Active Request Count", "requestTiming.activeRequestCount.description", GAUGE, MetricUnits.NONE, "ActiveRequestCount", null, null },
+			{ "vendor", "requestTiming.slowRequestCount", "Slow Request Count", "requestTiming.slowRequestCount.description", GAUGE, MetricUnits.NONE, "SlowRequestCount", null, null },
+			{ "vendor", "requestTiming.hungRequestCount", "Hung Request Count", "requestTiming.hungRequestCount.description", GAUGE, MetricUnits.NONE, "HungRequestCount", null, null }
+		};
+		mappingTable.put("WebSphere:type=RequestTimingStats,name=*", requestTimeTable);
+
+
+		String[][] threadPoolTable = new String[][] {
+			{ "vendor", "threadpool.activeThreads", "Active Threads", "threadpool.activeThreads.description", GAUGE, MetricUnits.NONE, "ActiveThreads", null, THREADPOOL_TAG_NAME },
+			{ "vendor", "threadpool.size", "Thread Pool Size", "threadpool.size.description", GAUGE, MetricUnits.NONE, "PoolSize", null, THREADPOOL_TAG_NAME }
+		};
+		mappingTable.put("WebSphere:type=ThreadPoolStats,name=*", threadPoolTable);
+
+		String[][] servletTable = new String[][] {
+        	{ "vendor", "servlet.request.total", "Total Request", "servlet.request.total.description", COUNTER, MetricUnits.NONE, "RequestCount", null, SERVLET_TAG_NAME },
+        	{ "vendor", "servlet.responseTime.total", "Total Response Time", "servlet.responseTime.total.description", GAUGE, MetricUnits.NANOSECONDS, "ResponseTimeDetails", "total", SERVLET_TAG_NAME }
+        };
+		mappingTable.put("WebSphere:type=ServletStats,name=*", servletTable);
+		
+		
+//		String[][] restTable = new String[][] {
+//			{ "base", "REST.request", "Total Requests and Response Time", "REST.request.description", SIMPLE_TIMER, MetricUnits.NANOSECONDS, "RequestCount", null, null, "ResponseTimeDetails", "total" }
+//		};
+//
+//		mappingTable.put("WebSphere:type=REST_Stats,name=*", restTable);
```

Comment:
If there is a reason to keep this commented out code please add a comment.  Otherwise delete it.